src/public/interfaces.ts

```diff
@@ -466,3 +466,19 @@ export interface LoadContext {
    */
   contentUrl: string;
 }
+
+/**
+ * @private
+ * Hide from docs.
+ */
+export interface FrameContext {
+  /**
+   * The current URL that needs to be used in the iframe if the tab is reloaded
+   */
+  contentURL: string;
```

Comment:
NIT: Rename `contentURL` to `contentUrl` to match the naming we use in the `Settings` interface